project/plugins.sbt

```diff
@@ -1 +1 @@
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.0")
\ No newline at end of file
+addSbtPlugin("io.github.gitbucket" % "sbt-gitbucket-plugin" % "1.2.0")
```

Comment:
Now, you can use sbt-gitbucket-plugin 1.3.0. It adds GitBucket version to assembly jar filename.

Reply:
I already bumped the version after the merge, but thanks for the hint.